style.css

```diff
@@ -0,0 +1,286 @@
+@font-face {
+  font-family: Cocogoose;
+  src: url('../assets/fonts/cocogoose/Cocogoose Pro-trial.ttf');
+}
+
+html,
+body,
+input,
+button,
+ul,
+li,
+a {
+  margin: 0;
+  border: 0;
+  padding: 0;
+  font: inherit;
+  text-align: inherit;
+  text-decoration: inherit;
+  color: inherit;
+  background: transparent;
+  list-style: none;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+h1 {
+  font-size: 5vw;
+}
+
+h2,
+h3 {
+  font-family: 'Cocogoose', sans-serif;
+  font-size: 3vw;
+}
+
+img {
+  width: 100px;
+}
+
+a {
+  cursor: pointer;
+}
+
+a:hover {
+  color: #ec5242;
+}
+
+a:active {
+  color: #ec5242;
+}
+
+p {
+  color: #272a31;
+  font-weight: 100;
+}
+
+.row {
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.cta {
+  font-size: 1.2rem;
+}
+
+.hr-title {
+  border: 1px solid #ec5242;
+  width: 10%;
+}
+
+.hr-speakers {
+  border: 1px solid #3c3e43;
+  width: 10%;
+}
+
+.section-title {
+  font-size: 1.5rem;
+}
+
+.section-subtitle {
+  font-size: 1.6rem;
+}
+
+.logo {
+  padding-top: 4px;
+  padding-bottom: 4px;
+}
+
+.logo img {
+  border-radius: 8px;
+}
+
+.image-effect {
+  background-image: linear-gradient(rgba(236, 82, 66, 0.5), rgba(236, 82, 66, 0.5)), url("./images/red.jfif");
+  background-repeat: repeat;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  text-transform: uppercase;
+  -webkit-font-smoothing: antialiased;
+}
+
+.nav-small {
+  font-size: 3rem;
+  top: 15%;
+  left: 5%;
+}
+
+.campaign {
+  border: 5px solid #ec5242;
+}
+
+.slider {
+  background: #f8f8f8;
+  background-image: linear-gradient(rgba(248, 248, 248, 0.7), rgba(248, 248, 248, 1)), url("./images/kaap.jfif");
+  background-size: cover;
+}
+
+.slider-about {
+  background: #f8f8f8;
+  background-image: linear-gradient(rgba(248, 248, 248, 0.7), rgba(248, 248, 248, 1));
+  background-size: cover;
+}
+
+.red {
+  color: #ec5242;
+}
+
+.bg-red {
+  background-color: #ec5242;
+}
+
+.bg-dark-light-custom {
+  background-color: rgba(143, 143, 143, 0.5);
+}
+
+.bg-dark-custom {
+  background-color: #272a31;
+}
+
+.welcome-description {
+  background-color: #f7f7f7;
+  border: 3px solid rgb(255, 255, 255);
+  padding: 20px;
+  margin: 20px 0;
+  font-weight: var(--weight);
+  font-size: 0.99rem;
+  line-height: 30px;
+}
+
+.main-program {
+  background-image: linear-gradient(rgba(40, 43, 50, 0.9), rgba(40, 43, 50, 0.9)), url("https://images.assetsdelivery.com/compings_v2/floralset/floralset1905/floralset190500161.jpg");
+  background-size: cover;
+}

```

Comment:
- [ ] The main-program section for the mobile version has its elements organized in a horizontal row, with the image at the left, the title in the center, and the description at the right also make sure you use the same color, font,... Please kindly fix that 😉

Your design
<img width="404" alt="Screen Shot 2022-12-02 at 15 53 21" src="https://user-images.githubusercontent.com/74501523/205298917-03120e13-1c6f-4101-9003-15b38ad44755.png">

Original design
<img width="455" alt="Screen Shot 2022-12-02 at 15 53 44" src="https://user-images.githubusercontent.com/74501523/205298971-0c9ef578-4465-48c2-b76e-eb098342b7b8.png">